docs/other-components/self_hosted_llms/index.md

```diff
@@ -4,6 +4,6 @@ AAQ allows you to connect to self-hosted LLMs instead of using OpenAI LLMs. This
 
 ## :question: How does it work?
 
-AAQ uses LiteLLM to connect to both OpenAI's API as well as self-hosted LLMs. By default, we use OpenAI endpoints, however, the user can switch to any LiteLLM supported hosted model by changing the `LLM_MODEL` and `LLM_ENDPOINT` environment variable in the `.env` file to their respective values.
+AAQ uses LiteLLM to connect to both OpenAI's API as well as self-hosted LLMs. By default, we use OpenAI endpoints, however, the user can switch to any LiteLLM supported hosted model by changing the `LITELLM_MODEL_DEFAULT` and `LITELLM_ENDPOINT_DEFAULT` environment variable in the `.env` file to their respective values.
```

Comment:
Can we also mention that they can choose different models for different tasks and that the details can be found on app/config.py? :D